nestkernel/eprop_archiving_node.cpp

```diff
@@ -0,0 +1,300 @@
+/*
+ *  eprop_archiving_node.cpp
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+// nestkernel
+#include "eprop_archiving_node.h"
+#include "kernel_manager.h"
+
+// sli
+#include "dictutils.h"
+
+namespace nest
+{
+
+nest::EpropArchivingNode::EpropArchivingNode()
+  : ArchivingNode()
+  , gamma_( 0.3 )
+{
+}
+
+nest::EpropArchivingNode::EpropArchivingNode( const EpropArchivingNode& n )
+  : ArchivingNode( n )
+  , gamma_( n.gamma_ )
+{
+}
+
+void
+nest::EpropArchivingNode::get_status( DictionaryDatum& d ) const
+{
+  ArchivingNode::get_status( d );
+
+  def< double >( d, names::gamma, gamma_ );
+}
+
+void
+nest::EpropArchivingNode::set_status( const DictionaryDatum& d )
+{
+  ArchivingNode::set_status( d );
+
+  double new_gamma = gamma_; // preserve values for the case invalid values are set
+  updateValue< double >( d, names::gamma, new_gamma );
+  gamma_ = new_gamma;
```

Comment:
`gamma` was removed in https://github.com/nest/nest-simulator/pull/2867/commits/045afd6faea234bd83a01931ee4ab7bd0b71cef3 and since it was the only variable of `EpropArchivingNode` the `set_status` and `get_status` functions were also removed